ultraplot/axes/plot.py

```diff
@@ -3666,6 +3724,143 @@ def fill_betweenx(self, *args, **kwargs):
         kwargs = _parse_vert(default_vert=False, **kwargs)
         return self._apply_fill(*args, **kwargs)
 
+    @docstring._snippet_manager
+    def graph(
+        self,
+        g: Union["nx.Graph", np.ndarray],
+        layout=None,
+        layout_kw={},
+        node_kw={},
+        edge_kw={},
+        label_kw={},
+        labels: Union[None, bool, Iterable] = None,
+        nodes: Union[None, bool, Iterable] = None,
+        edges: Union[None, bool, Iterable] = None,
+        grid: Union[None, str] = None,
+        aspect: Union[None, str] = None,
+        facecolor: Union[None, str] = None,
+        spines: Union[None, bool, Iterable, str] = None,
+        rescale: Union[None, bool] = None,
```

Comment:
These keywords should be removed.

Reply:
Are you referring to the typehinting? I personally like them.

Reply:
No the actual function keywords. It's odd to supply this small subset of apis here when many other functions do not.

Reply:
I see what you mean. However, most functions override matplotlib functionality and so keywords are passed on. When I compare UltraPlot native fuctions (e.g. `format` there are keywords given. In this case there is no matplotlib equivalent.  I would lean to having them in over having `*args, **kwargs`; or is there a way to sanitize the input with allowed keywords?

Reply:
My point was that people should be using format as the primary way to format the plot. The keywords to the graph function should only concern the graph nodes and edges themselves. Right now that function has keywords that style the entire plot.

Reply:
Hmm. Ok we could be move the settings  to format and then have some defaults through `rc`, would that work? Then it would be more inline with "nice defaults but full control".

Reply:
Yeah that's totally fine by me.

Reply:
Ok let's go with that 😬

Reply:
Do the latest changes resolve this?